api4/system.go

```diff
@@ -68,6 +69,8 @@ func (api *API) InitSystem() {
 	api.BaseRoutes.System.Handle("/notices/view", api.ApiSessionRequired(updateViewedProductNotices)).Methods("PUT")
 
 	api.BaseRoutes.System.Handle("/support_packet", api.ApiSessionRequired(generateSupportPacket)).Methods("GET")
+	api.BaseRoutes.ApiRoot.Handle("/first_admin_visit_marketplace", api.ApiHandler(setFirstAdminVisitMarketplaceStatus)).Methods("POST")
+	api.BaseRoutes.ApiRoot.Handle("/first_admin_visit_marketplace", api.ApiSessionRequired(getFirstAdminVisitMarketplaceStatus)).Methods("GET")
```

Comment:
Should these be part of the Marketplace routes? While the code makes use of the system store the API still has to do with more the Marketplace then with the system.

Reply:
@hanzei, @jfrerich  - are you guys proposing having a `marketplace` prefix, like `/marketplace/first_admin_visit` or moving the handle statements to the `api4/plugin.go` file, or both?

Reply:
Right now all Marketplace related routes are in `api4/plugin.go`. I think the two new routes should be places in the same file. Given that the routes are under `/plugins/` right now, I leaning toward thinking `api4/plugin.go` is the right place.

Should the new routes be under `/plugins/marketplace/first_admin_visit` despite `/plugins/marketplace/` being an a handler?

Reply:
Don't think there is an issue with that.